src/styles/custom-components.css

```diff
@@ -59,6 +59,17 @@
     color: var(--black-600) !important;
 }
 
+.s-btn-group .s-btn--radio:checked + .s-btn {
```

Comment:
To keep things extra safe can we scope this rule by targeting `.s-btn-group .s-btn--radio:checked + .s-editor-btn` instead? Sane for the rule below